PhpcsChanged/Cli.php

```diff
@@ -341,20 +348,25 @@ function runGitWorkflowForFile(string $gitFile, array $options, ShellOperator $s
 	$phpcsStandard = $options['standard'] ?? null;
 	$phpcsStandardOption = $phpcsStandard ? ' --standard=' . escapeshellarg($phpcsStandard) : '';
 
+	$warningSeverity = $options['warning-severity'] ?? null;
+	$phpcsStandardOption .= isset($warningSeverity) ? ' --warning-severity=' . escapeshellarg($warningSeverity) : '';
+	$errorSeverity = $options['error-severity'] ?? null;
+	$phpcsStandardOption .= isset($errorSeverity) ? ' --error-severity=' . escapeshellarg($errorSeverity) : '';
```

Comment:
Related to the last comment, these two ternaries use `isset()`  but we may want to catch empty strings also? It doesn't matter much but let's make it the same as the above whichever we choose.